cardano-rt-view/src/Cardano/Benchmarking/RTView/NodeState/Updater.hs

```diff
@@ -97,6 +95,11 @@ updateNodesState nsMVar loggerName (LogObject aName aMeta aContent) = do
       Just ns ->
         if | itIsErrorMessage aMeta ->
               nodesStateWith $ updateNodeErrors ns aMeta aContent
+           | "cardano.node.metrics.peersFromNodeKernel" `T.isInfixOf` aName ->
```

Comment:
please make sure that the node correctly routes these messages only to the forwarder.